src/stcal/ramp_fitting/ols_fit.py

```diff
@@ -22,7 +22,7 @@
 
 
 def ols_ramp_fit_multi(
-        input_model, buffsize, save_opt, readnoise_2d, gain_2d, weighting, max_cores):
+        ramp_class, buffsize, save_opt, readnoise_2d, gain_2d, weighting, max_cores):
```

Comment:
I think having an instance called "ramp_class" is a bit confusing. I know it is an instance of a class, but that's always the case with python objects.  It would be better to not have "class" in the name.  Maybe `ramp_data`? Or something similar?

Reply:
I changed all instances of "ramp_class" as a name to "ramp_data".